core/src/index/thirdparty/faiss/gpu/impl/L2Select.cu

```diff
@@ -195,7 +55,7 @@ __global__ void l2SelectMin1(Tensor<T, 2, true> productDistances,
     for (int row = rowStart; row < productDistances.getSize(0); ++row) {
       for (int col = threadIdx.x; col < productDistances.getSize(1);
            col += blockDim.x) {
-        if (!(bitset[col >> 3] & (0x1 << (col & 0x7)))) {
+        if ((bitset.getSize(0) == 0) || (!(bitset[col >> 3] & (0x1 << (col & 0x7))))) {
```

Comment:
(bitset.getSize(0) == 0) should be moved out of for loop

Reply:
fixed